src/steamship/cli/cli.py

```diff
@@ -758,6 +759,43 @@ def support_info():
     click.secho(f"In virtual env: {sys.prefix != sys.base_prefix}")
 
 
+@click.command()
+@click.option(
+    "--all",
+    is_flag=True,
+    default=False,
+    help="Delete all local development instances without prompting",
+)
+def delete_local_dev_instances(all: Optional[bool] = False):
+    client, _, manifest = initialize_and_get_client_and_prep_project()
+
+    if manifest.type != DeployableType.PACKAGE:
+        click.secho("May only delete development instances of Packages.", fg="red")
+        return
+
+    click.secho(
+        f"Deleting LOCAL DEVELOPMENT instances of package [{manifest.handle}] across workspaces."
+    )
+    for instance in PackageInstance.list(
+        client, include_workspace=True, across_workspaces=True
+    ).package_instances:
+        # Only consider deletion for LOCAL DEVELOPMENT instances of THIS PACKAGE.
+        if (
+            instance.package_version_handle == LOCAL_DEVELOPMENT_VERSION_HANDLE
+            and instance.package_handle == manifest.handle
+        ):
+            workspace = Workspace.get(client, instance.workspace_id)
+            result = all or click.confirm(
+                f"Delete instance [{instance.handle}] in workspace [{workspace.handle}] created {instance.created_at}"
+            )
+            if result:
+                instance.delete()
```

Comment:
try/except here, to allow deletion to continue?

Reply:
Should it continue, if it can't delete something?

Reply:
This strikes me as a "I want to clean up as much as possible" thing, I'd personally be annoyed if this failed for some reason on the first one and I had 20 other files that needed deleting.

Reply:
👍